### ISSUE TITLE
Support for FiveCrop/TenCrop and frame rate conversion

### ISSUE THREAD

**Issue by borijang** [v-iashin/video_features#72]
Hey, thanks for this repo.

I noticed a lot of architectures use five-cropped or ten-cropped features, so I am wondering if you considered/would consider implementing this feature.
Additionally, I am currently converting the frame rate of the videos before extracting the features, but I think it would be great if we could do both at once.

**Comment by v-iashin** [comment]
Yep, this would be nice to have. Do you mean the 5-crop and 10-crop features only for frame-wise feature extractors? Is there an equivalent for video?

Regarding the second part, among other reasons, I decided to re-encode it first to a file and then load it because loading the whole video with the original fps might be hurtful for the user's RAM. It is also the way how one would do it when pre-processing the dataset. Therefore, it is more convenient to do it separately for now. 

A large number of corner cases when doing the conversion on the fly makes it unattractive for implementation within this library and I would rather rely on something popular and well-tested (e.g. how to make 17 fps video from 30 fps original). 

We also currently discussing this and video IO in general in #61 You are welcome to participate in the discussion.

**Comment by borijang** [comment]
Thanks for the incredibly fast response!

Yep, specifically for me I'd use FiveCrop / TenCrop on the I3D model.

Regarding the fps conversion, I somehow missed the `extraction_fps` argument and `utils/reencode_video_with_diff_fps`, I think that's exactly what I wanted.

**Comment by v-iashin** [comment]
Great! I will keep this opened.

If someone else would like to see FiveCrop / TenCrop implemented, let me know in this issue and I will take a look!

**Comment by v-iashin** [comment]
i don't have the resources to implement it myself but i'd love to consider a pull request implementing this (fixing or redoing the earlier attempt: #110).
